language-server/vscoqtop/lspManager.ml

```diff
@@ -211,6 +216,23 @@ let coqtopStepForward ~id params : (string * Dm.DocumentManager.events) =
   Hashtbl.replace states uri st;
   update_view uri st;
   (uri,events)
+  
+  let make_label (label, typ, path) = 
```

Comment:
How about defining this type (in `LspData`) and letting `yojson` do the encoding? The "Path" concatenation could be done outside the encoding part, for better separation of concerns.

Reply:
Yeah, thats a better idea. I will change it later today